setup/providers/airvpn.md

```diff
@@ -1,18 +1,22 @@
 # AirVPN
 
-## TLDR
+## Wireguard
 
-### Wireguard
+1. [Generate and download your .conf file](https://airvpn.org/generator/) - Select WireGuard, continents and hit *Generate* at the bottom
+1. Open the newly generated .conf file anad take note of the `[Interface] PrivateKey`, `[Peer] PresharedKey` and `[Interface] Address`
+1. Launch docker image with:
```

Comment:
I would rather keep the sections structure and the examples as before, but add as bullet points right below `### Wireguard`:

- ➡️ [generate your Wireguard config](https://airvpn.org/generator/) 
- Only set the first IPv4 address in `WIREGUARD_ADDRESSES`, unless you have IPv6 working on Docker

The rest is rather obvious to the user I believe and doesn't need to be mentioned 🤔